Libraries/BatchedBridge/MessageQueue.js

```diff
@@ -63,6 +63,7 @@ class MessageQueue {
     this._callID = 0;
     this._lastFlush = 0;
     this._eventLoopStartTime = new Date().getTime();
+    this._immediatesCallback = null;
```

Comment:
Can you also add a Flow type for the member variable?

Should be able to add this to below line 50:

    _immediatesCallback: ?() => void;